Gordon360/Controllers/AdvancedSearchController.cs

```diff
@@ -136,5 +136,20 @@ public ActionResult<IEnumerable<string>> GetBuildings()
             return Ok(buildings);
         }
 
+        /// <summary>
+        /// Return a list involvements.
```

Comment:
if you could give us a slightly more specific (verbose) summary, that would be best. Because it technically returns a list of involvement descriptions (which should also require a slightly better function name)